README.md

```diff
@@ -23,3 +23,13 @@ inefficient.
 
 What is the big $\Theta$ complexity of your implementation? Add your
 answer, including your reasoning, to this markdown file.
+
+# Solution
+
+First I initialize two arrays that are the size of the number of vertices (n), taking O(n). The enqueue method iterates over the elements and finds the correct index (according to its priority) to place the value, this means that it takes linear time, O(n). However, the total number of enqueue calls is O(m) where m is the number of edges. This means that our overall complexity is $\Theta(n \cdot n \cdot m) = \Theta(n^2 \cdot m)$ which (assuming very large connected graph) simplifies to $\Theta(n^3)$.
```

Comment:
Don't see why you assume a very large graph here. It looks like you're assuming that the number of edges in linear in the number of nodes?

Reply:
Up to you, but if you make any assumptions they have to be justified.